src/visualizers/ModFill.ts

```diff
@@ -81,8 +80,4 @@ class VizModFill extends P5Visualizer {
     }
 }
 
-export const exportModule = new VisualizerExportModule(
-    'Mod Fill',
-    VizModFill,
-    ''
-)
+export const exportModule = new VisualizerExportModule(ModFill)
```

Comment:
the other vis's have a description at the export line.  For this one a reasonable desc would be 'A dynamic histogram of modular values.' (If I'm recalling the concept correctly.)

Reply:
OK. The description is optional, and is unused in the current app, but it could be like a tooltip or something in a revamped UI. The fact there is not one here just reflects that there was not one before for ModFill.  But I totally get how this might be good juncture to fill all the rest in. I will add a commit doing that. Do you also want me to make the description mandatory?